runtime/runtime.m

```diff
@@ -2300,6 +2319,7 @@ -(void) xamarinSetFlags: (enum XamarinGCHandleFlags) flags;
 		mode = xamarin_is_gc_coop ? MarshalManagedExceptionModeThrowObjectiveCException : MarshalManagedExceptionModeUnwindNativeCode;
 
 	switch (mode) {
+#if !defined (CORECLR_RUNTIME) // CoreCLR won't unwind through native frames, so we'll have to abort (in the default case statement)
```

Comment:
should be added to https://github.com/xamarin/xamarin-macios/issues/10795

Reply:
Added as _It's not possible to have CoreCLR unwind native frames._